quark-test.c

```diff
@@ -851,6 +905,7 @@ const struct test all_tests[] = {
 	T(t_cache_grace),
 	T(t_min_agg),
 	T(t_stats),
+	T(t_nada),
```

Comment:
```suggestion
```
right?

Reply:
woooooops, in my defense, it was late :dagger: 
many thanks!